html-forms/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="post"><h1>Coding Form!</h1></form>
+    <section>
+      <label for="email">Email</label>
+      <div>
+        <input type="email" id="email" name="email" />
+      </div>
+    </section>
+
+    <section>
+      <label for="select">Why is coding cool?</label>
+      <div>
+        <select id="select" name="select">
+          <option value="option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </section>
+    <section>
+      <p>
+        <label for="radio">Favorite Language (choose one)</label>
+      </p>
+
+      <div>
+        <label for="HTML">
+          <input type="radio" id="HTML" name="radio" value="H" />
+          HTML
+        </label>
+        <label for="CSS">
+          <input type="radio" id="CSS" name="radio" value="C" />
+          CSS
+        </label>
+        <label for="JS">
+          <input type="radio" id="JS" name="radio" value="J" />
+          JavaScript
+        </label>
+      </div>
+    </section>
+    <section>
+      <p>
+        <label for="checkbox"
+          >Would you like to sign up for more information?</label
+        >
```

Comment:
This text should just go inside the `p` tag. You already have another label associated with the `input` on line 56